prime-router/src/main/kotlin/fhirengine/engine/LookupTableValueSet.kt

```diff
@@ -27,8 +27,8 @@ data class LookupTableValueSetConfig(
  */
 class LookupTableValueSet
     (@JsonProperty("lookupTable") private val configData: LookupTableValueSetConfig) : ValueSetCollection {
-    private val metadata = Metadata.getInstance()
     private val mapVal: SortedMap<String, String> by lazy {
+        val metadata = Metadata.getInstance()
```

Comment:
I dropped this into the `lazy` block to avoid starting up the whole database connection pool during the validation step where all I care about is the parsed values from YAML.